src/module.rs

```diff
@@ -14,12 +14,14 @@ use tracer::{EntryKind, EXPOSE_TRACER, EXPOSE_TRACER_LEN, LOG_CALL};
 static VOID_VALUE_PLACEHOLDER: i32 = i32::MAX;
 
 #[derive(Debug)]
+/// WebAssembly module
```

Comment:
Maybe change this to "wrapper around the parity-wasm `Module` struct, with convenience functions"?